src/NotificationEventMailing.php

```diff
@@ -518,10 +514,7 @@ private static function attachDocuments(Email $mail, array $documents_ids)
             }
             $path = GLPI_DOC_DIR . "/" . $document->fields['filepath'];
             if (Document::isImage($path)) {
-                $path = Document::getImage(
-                    $path,
-                    'mail'
-                );
+                $path = Document::getImage($path);
             }
```

Comment:
This whole piece of code can be removed.

Reply:
done !